generate/generate.go

```diff
@@ -604,6 +598,12 @@ func (g *Generator) SetLinuxResourcesMemorySwappiness(swappiness uint64) {
 	g.spec.Linux.Resources.Memory.Swappiness = &swappiness
 }
 
+// SetLinuxResourcesDisableOOMKiller sets g.spec.Linux.Resources.Memory.DisableOOMKiller.
+func (g *Generator) SetLinuxResourcesDisableOOMKiller(disable bool) {
```

Comment:
Do you want to use `SetLinuxResourcesMemoryDisableOOMKiller` to match [these][1]?

[1]: https://github.com/opencontainers/runtime-tools/blob/6210a304b5ef728abbd40f0d6ccf61e1a83e44fc/generate/generate.go#L571-L599